dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -19,6 +19,10 @@ RUN apt-get install -y curl ca-certificates
 # Install gcc which is required for installing hiredis
 RUN apt-get install -y gcc
 
+# Install libdpkg-perl which is required for python3.6-3.6.0 as one of its specs i.e. no-pie-compile.specs
+# The file referenced (`/usr/share/dpkg/no-pie-compile.specs`) is in the `libdpkg-perl` package on Debian
+RUN apt-get install -y libdpkg-perl
```

Comment:
>libdpkg-perl [](start = 23, length = 12)

Just wondering why it is not needed in previous Jessie image?

Reply:
Is it only need to build python or library? Can we remove it after the build?

---
In reply to: [261457115](https://github.com/Azure/sonic-buildimage/pull/2620#discussion_r261457115) [](ancestors = 261457115)

Reply:
Yes. You can remove it after the build.

Reply:
Yes. We can remove it after the build.

Reply:
This comment is still applicable.

---
In reply to: [269791287](https://github.com/Azure/sonic-buildimage/pull/2620#discussion_r269791287) [](ancestors = 269791287,261457115)

Reply:
@qiluo-msft , it's needed during snmp docker build.  Otherwise, I get error like below
 ->
 ```x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fdebug-prefix-map=/sonic/src/python3/python3.6-3.6.0=. -specs=/usr/share/dpkg/no-pie-compile.specs -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=561 -DPSUTIL_LINUX=1 -I/usr/include/python3.6m -c psutil/_psutil_common.c -o build/temp.linux-x86_64-3.6/psutil/_psutil_common.o
    x86_64-linux-gnu-gcc: error: /usr/share/dpkg/no-pie-compile.specs: No such file or directory
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

    ----------------------------------------
Command "/usr/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-3892p393/psutil/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-p9uabxzp/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-3892p393/psutil/
The command '/bin/sh -c pip install /python-wheels/asyncsnmp-2.1.0-py3-none-any.whl' returned a non-zero code: 1
[  FAIL LOG END  ] [ target/docker-snmp-sv2.gz ]
slave.mk:489: recipe for target 'target/docker-snmp-sv2.gz' failed
make: *** [target/docker-snmp-sv2.gz] Error 1```